client/app/assets/index.html

```diff
@@ -8,7 +8,6 @@
     <meta name="description" content="Jenkins Pipeline">
     <link rel="icon" type="image/png" href="images/favicon.png">
 
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
This line just loads the materialize icons not the entire framework. And as
it has a poor variety of icons (and doesn't have the one I needed) I
decided to go with font awesome.

Junior Ales
http://juniorales.com

(enviado pelo meu andróide)
Em 2 de fev de 2016 23:16, "Vinícius Gama" <notifications@github.com>
escreveu:

> In client/app/assets/index.html
> <https://github.com/jenkins-pipeline/jenkins-pipeline/pull/21#discussion_r51665357>
> :
>
> > @@ -8,7 +8,6 @@
> >      <meta name="description" content="Jenkins Pipeline">
> >      <link rel="icon" type="image/png" href="images/favicon.png">
> >
> > -    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
>
> why did you removed this? not using material from google anymore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkins-pipeline/jenkins-pipeline/pull/21/files#r51665357>
> .
>